docs/syscalls.md

```diff
@@ -1,10 +1,14 @@
-# Home of Falco drivers syscalls report
+# Home of Falco Drivers Syscalls Report
 
-Thanks to our [syscalls-bumper](https://github.com/falcosecurity/syscalls-bumper) project, we are able to always support latest syscalls added to linux kernel.  
-Support for new syscalls is initially automatically added by the tool as generic events; when needed, a generic event can be made "specific",  
-by creating a whole new event to track it.
+Thanks to our [syscalls-bumper](https://github.com/falcosecurity/syscalls-bumper) project, we are able to always support latest syscalls added to the Linux kernel.
+
+Our automation adds new syscalls as generic events (you can find the definition below to better understand what this term means within our libs). If necessary, developers can create new parsers to extract and make available each syscall argument. Internally, we refer to the extractors on the kernel side as "fillers". This process also involves the creation of a new event type in libsinsp.
 
 ## Glossary
 
-* 🟢 -> means that the syscall is implemented as a specific event
-* 🟡 -> means that the syscall is implemented as a generic event
+The Falco Projects' kernel drivers support monitoring a range of syscalls. For a subset of syscalls (indicated with 🟡), we only monitor when the syscalls are triggered (internally in libs, we refer to these as generic syscalls), but we do not extract the syscall arguments for those.
```

Comment:
Is this part a leftover?
It is lready present in the index page :)

Reply:
🤦‍♀️ it was intentional, ok tried to give it a whole new spin, let me know what you think about the new version. 

Apart from the language edits I mostly hope these changes may make the navigation a bit easier.

Lastly, folks may just directly navigate to the supported syscalls table, why not put the Glossary right there?

Reply:
> Lastly, folks may just directly navigate to the supported syscalls table, why not put the Glossary right there?

Because tables are autogenerated thus it is much harder to make changes like the one you are doing right now (it would require GO source file changes to syscalls-bumper in the syscalls table case, and to kernel-testing repo for the kernel matrixes).

Reply:
I think it is better to leave glossaries to the opening page, and then leave the autogenerated pages on their own.

Reply:
ACK, changed it.